content/webapp/components/Exhibition/Exhibition.tsx

```diff
@@ -554,13 +538,10 @@ const Exhibition: FunctionComponent<Props> = ({
       // We hide contributors as we show them further up the page
       hideContributors={true}
     >
-      {exhibitionAccessContent && exhibition.uid !== 'being-human' ? (
+      {exhibition.uid !== 'being-human' ? (
```

Comment:
Why is this there? Should we add a comment?

Reply:
this was a very good question, and one that I didn't address (and one that I'm now wondering about)

Reply:
https://github.com/wellcomecollection/wellcomecollection.org/issues/11732